config/application.rb

```diff
@@ -49,5 +49,11 @@ class Application < Rails::Application
                        request_specs: false
       g.fixture_replacement :factory_girl, dir: 'spec/factories'
     end
+
+    # For API Cross Origin Access
+    config.action_dispatch.default_headers = {
+      'Access-Control-Allow-Origin' => '*',
+      'Access-Control-Request-Method' => %w{GET POST OPTIONS}.join(",")
```

Comment:
__[rubocop]__
* Style/TrailingCommaInLiteral: Put a comma after the last item of a multiline hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)
* Style/PercentLiteralDelimiters: `%w`-literals should be delimited by `(` and `)`. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-literal-braces)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->